autoload/neoterm/repl/python.vim

```diff
@@ -0,0 +1,67 @@
+"init python specific settings
+if !exists('g:neoterm_repl_ipy_magic')
+  let g:neoterm_repl_ipy_magic = 0
+end
+
+function! neoterm#repl#python#is_valid(value)
```

Comment:
I thought we would _just_ validate if the given command is valid, in the same sense that already exists, if the executable exists. I was thinking in something like:
```vimscript
function! neoterm#repl#python#is_valid(value)
  let l:cmd =  type(a:value) == v:t_list ? join(a:value) : a:value

  if l:cmd =~# '\<iphython\>'
    return executable('ipython')
  elseif l:cmd =~# '\<python\>'
    return executable('python')
  else
    return false
  end
endfunction

Reply:
Thank you, i've already searched for two things you've done on your code.

My code is currently doing a little more. Every command in the list is checked and ipython is automatically extended by "--no-autoindent". The last one was already present. Is that no longer necessary?

Reply:
does `ipython` require the `--no-autoindent`? If so I think we should document that instead of changing the user input.

Reply:
Yeah, i guess it would be better to append the documentation.
Should i try to append the docs?
I'm no native speaker so it will need some corrections ;)

Reply:
`executable` checks if the given value exists in the environment and if it's an executable (bin).

Reply:
> Should i try to append the docs?

yes.


> I'm no native speaker so it will need some corrections ;)

that's fine.

Reply:
Ok